.github/workflows/ports_windows.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       with:
         vs-version: ${{ matrix.vs_version }}
     - uses: actions/setup-python@v5
-      if: matrix.runner == 'windows-2019'
+      if: matrix.runner == 'windows-2025'
       with:
         python-version: '3.9'
```

Comment:
Last time I checked, windows-2025 still uses python 3.9 by default, so we can just drop this whole section if we are dropping use of the windows-2019 runner. This was only needed because windows-2019 used an older python version by default.